.travis.yml

```diff
@@ -20,19 +20,22 @@ notifications:
 node_js:
   - '8'
 env:
-  - TASK=lint
-  - TASK=test:finance
-  - TASK=test:survey
-  - TASK=test:token-manager
-  - TASK=test:token-manager:app
-  - TASK=test:vault
-  - TASK=test:voting
-  - TASK=coverage:finance
-  - TASK=coverage:survey
-  - TASK=coverage:token-manager
-  - TASK=coverage:vault
-  - TASK=coverage:voting
-  - TASK=test:shared
+  global:
+    - INSTALL_FRONTEND=false
+  matrix:
+    - TASK=lint
+    - TASK=test:finance
+    - TASK=test:survey
+    - TASK=test:token-manager
+    - TASK=test:token-manager:app INSTALL_FRONTEND=true
```

Comment:
I think this can be removed?

Reply:
`INSTALL_FRONTEND` needs to be `true` in this case as `test:token-manager:app` actually does test the frontend part (if you don't install the frontend then you get the error https://travis-ci.org/danielcaballero/aragon-apps/jobs/489963180)

Unless you meant removing this line all together? Let me know.

Reply:
Ahh gotcha :+1:. Forgot there were tests in just the Token Manager's frontend.